test/observe_test.dart

```diff
@@ -106,6 +108,7 @@ void _observeTests(createModel(x)) {
 
     subs.add(t.changes.listen((n) {}));
     expect(t.hasObservers, true);
+    Observable.dirtyCheck();
```

Comment:
I see - any clues on what's the issue? It might be worth waiting until it is fixed